pages/hub/apps/guides/debridge.mdx

```diff
@@ -0,0 +1,301 @@
+import { Callout } from 'components'
+
+# deBridge cross-chain bets
+
+The [deBridge](https://docs.dln.trade/dln-api/quick-start-guide) provides the way to execute user's bet transaction in `Polygon` from another blockchain, for example `Arbitrum`.
+
+<Callout type="warning">
+  This feature works only for `Prematch Bets` in [`Supported Blockchains`](https://docs.dln.trade/the-core-protocol/fees-and-supported-chains)
+</Callout>
+
+First of all, you need list of supported blockchains and tokens
```

Comment:
First of all, you need to get a list of supported blockchains and tokens

Reply:
f13e4127dec44bcbf66bbd1bc0b1418ef32f9a51